gbdxtools/vectors.py

```diff
@@ -399,16 +400,19 @@ def map(self, features=None, query=None, styles=None,
         })
         template.inject()
 
-    def _build_image_layer(self, image, image_bounds):
+    def _build_image_layer(self, image, image_bounds, cmap):
         if image is not None:
             if isinstance(image, da.Array):
-                arr = image.rgb()
+                if len(image.shape) == 2:
```

Comment:
Will this work for the case in gbdxtools where the single band image shape is `(1, 1000, 1000)`?

Reply:
I wasn't sure if that ever happens, single band slices seem to be (r,c), not (r,c,1)

Reply:
Handling this now just in case